src/presets.js

```diff
@@ -119,4 +119,9 @@ module.exports = {
       showNotFound: true,
     },
   },
+  babel: {
+    System: ['OS'],
```

Comment:
I remove `CPU` information because
1) From my experience of triaging babel issues, CPU is not much relevant
2) It is fairly easy, especially for Apple products, to guess the exact machine model from the CPU name. I think the machine model should be consider privacy and thus should not reveal by default.